x-pack/plugins/observability_solution/slo/public/ui_actions/index.ts

```diff
@@ -29,7 +31,11 @@ export function registerSloUiActions(
   // Assign triggers
   uiActions.addTriggerAction(CONTEXT_MENU_TRIGGER, editSloAlertsPanelAction);
   uiActions.addTriggerAction(CONTEXT_MENU_TRIGGER, editSloOverviewPanelAction);
-  uiActions.addTriggerAction('ADD_PANEL_TRIGGER', addOverviewPanelAction);
-  uiActions.addTriggerAction('ADD_PANEL_TRIGGER', addErrorBudgetPanelAction);
-  uiActions.addTriggerAction('ADD_PANEL_TRIGGER', addAlertsPanelAction);
+
+  if (!isServerless) {
+    // only register these actions in stateful kibana
+    uiActions.addTriggerAction(ADD_PANEL_TRIGGER, addOverviewPanelAction);
+    uiActions.addTriggerAction(ADD_PANEL_TRIGGER, addErrorBudgetPanelAction);
+    uiActions.addTriggerAction(ADD_PANEL_TRIGGER, addAlertsPanelAction);
+  }
```

Comment:
exclude the registering of observability SLO actions in serverless.

Reply:
@eokoneyo Can you elaborate more why you had to exclude these actions on serverless? 

I have an open [PR](https://github.com/elastic/kibana/pull/186374), where I remove these custom actions in favor of the generic edit actions. After my PR is merged, does this mean that you could register SLO actions in serverless as well?

Reply:
Not registering the SLO edit actions on serverless would be a bit problematic.  We display a custom link within the SLO embeddable panels that opens the edit configuration. So in this case the link on serverless would not open anything and I think it will even show an error.

Reply:
Here's a Video of the custom link I mentioned https://github.com/elastic/kibana/assets/2852703/c609fa70-4c1f-4aa5-aa17-4e765456f7e6

Reply:
@eokoneyo I just noticed that this file is outdated and you indeed register these actions on stateful and observability serverless. In the beginning I thought you were disabling on serverless just the SLO `edit` actions, but that was my misunderstanding.

Can you explain what you mean by `In serverless, the observability panels (SLO*) only shows as a selection choice in the observability project type.`?